Kickstarter-iOS/Views/Cells/SettingsAccountWarningCell.swift

```diff
@@ -9,8 +9,11 @@ final class SettingsAccountWarningCell: UITableViewCell, ValueCell, NibLoading {
   @IBOutlet fileprivate weak var warningIconImage: UIImageView!
 
   func configureWith(value shouldHideAlertIcon: Bool) {
+    _ = self
+      |> \.accessibilityTraits .~ .button
```

Comment:
Since we only use this cell in one place this should be fine. Refactoring the signature would result in a lot more changes that don't seem necessary unless we start using this cell more.